packages/components/src/components/Badge/Badge.tsx

```diff
@@ -120,6 +127,8 @@ export const Badge = ({
             disabled={!!isDisabled}
             $hasAlert={!!hasAlert}
             className={className}
+            $margin={margin}
```

Comment:
nitpick: I think there is no need for custom margin property since you can apply later any styles with `Styled(Badge)`

Reply:
Extending components this way is not preferred (cc @jvaclavik) This is intentional, we want to be able to do everything we need without ever restyling a standard component.

Reply:
Exactly